docs/molgenis/use_usingpyclient.md

```diff
@@ -0,0 +1,281 @@
+
+| argument | type | description                                                                    | required | default |
+|----------|------|--------------------------------------------------------------------------------|----------|---------|
+| table    | str  | the name of a table                                                            | True     | None    |
+| schema   | str  | the name of a schema                                                           | False    | None    |
+| as_df    | bool | if true: returns data as pandas DataFrame <br/> else as a list of dictionaries | False    | False   |
+
+### export
+```python
+client.export(schema='My Schema', table='Data Table', fmt='csv')
+```
+Exports data from a schema to a file in the desired format.
+If the table is specified, only data from that table is exported, otherwise the export contains all tables on the schema.
+If all tables from a schema are exported with given format `csv`, the data is exported as a zip file containing a csv file of each table.
+Throws the `NoSuchSchemaException` if the user does not have at least _viewer_ permissions or if the schema does not exist.
+
+| argument | type | description                               | required | default |
+|----------|------|-------------------------------------------|----------|---------|
+| table    | str  | the name of a table                       | False    | None    |
+| schema   | str  | the name of a schema                      | False    | None    |
+| fmt      | str  | the output format, either `csv` or `xlsx` | False    | `csv`   |
+
+
+### save_schema
+```python
+client.save_schema(table='Data Table', name='My Schema',  
+                   file='location/of/data/file.csv', data=[{'col1': value1, ...}, {'col2': value2, ...}, ...])
+```
+Imports or updates records in a table of a named schema.
+The data either originates from a file on the disk, or is supplied by the user after, for example, preprocessing.
+Either `file` or `data` must be supplied. The data must be compatible with the schema to which it is uploaded. 
+Throws the `PermissionDeniedException` if the user does not have at least _editor_ permissions for this schema.
+Throws the `NoSuchSchemaException` if the schema is not found on the server.
+
+| argument | type | description                            | required | default |
+|----------|------|----------------------------------------|----------|---------|
+| table    | str  | the name of a table                    | True     |         |
+| schema   | str  | the name of a schema                   | False    | None    |
+| file     | str  | the location of a `csv` file with data | False    | None    |
+| data     | list | data as a list of dictionaries         | False    | None    |
+
+
+### delete_records
+```python
+client.delete_records(table='Data Table', name='My Schema',  
```

Comment:
For a later point:
What happens if users uses a file AND data?

Reply:
Same as `save_schema`